### ISSUE TITLE
Type "property"

### ISSUE THREAD

**Issue by shaedrich** [foundeo/cfdocs#718]
There are some properties which are just a sub to specific function or tag just like a member function is for a specific data type.

For example `columnList` or `recordcount`.

**Comment by pfreitag** [comment]
Yeah, it would be good to have a good way to document these... 

So if you think about a query object, that can be returned from a number of functions or tags, maybe we need to be able to document `query.json` , with stuff like this:

```
{
   name:"query",
   type: "object",
   params: [
      {name:"recordcount", description:"The total number of records"},
      ...
   ]
}
```

The problem is there is already a `query.json` so may have to call it `query-object.json` or something.

**Comment by alorne** [comment]
Why not `query-return.json`?

**Comment by shaedrich** [comment]
Or using another folder for meta data like data types.

**Comment by alorne** [comment]
Could something like so work.
`{"return-params": []}`

**Comment by shaedrich** [comment]
Hmmm&hellip; Does it (literally) "return" something?

**Comment by alorne** [comment]
true, then how about `result-params`? In ACF at least that is where you define the variable.

**Comment by shaedrich** [comment]
true as well^^
